tests/Behat/WebtoolsConfigContext.php

```diff
@@ -26,4 +26,19 @@ public function webtoolsAnalyicsConfigIsSet(string $id, string $sitepath): void
     $this->setConfig('oe_webtools_analytics.settings', 'sitePath', $sitepath);
   }
 
+  /**
+   * Backup configs that need to be reverted in AfterScenario.
+   *
+   * @BeforeScenario @BackupLacoConfigs
+   */
+  public function backupLacoConfigs() {
+
+    $name = 'oe_webtools_laco_widget.settings';
+
+    $configs = $this->getDriver()->getCore()->configGet($name);
```

Comment:
Since we're changing the value in an admin page, not through a behat step, I was avoiding to use the `setConfig()`. But ok, I use it and I give information in the comment.